challenges/Python_Parallel_HDF5/README.md

```diff
@@ -0,0 +1,230 @@
+# Python: Parallel HDF5
+
+Scientific simulations generate large amounts of data on Summit (about 100 Terabytes per day).
```

Comment:
I might say (about 100 Terabytes per day for some applications) since some do not generate this much. Nit-pickly, I know, but true :)